aspnetcore/tutorials/signalr.md

```diff
@@ -144,7 +141,7 @@ The SignalR server library is included in the `Microsoft.AspNetCore.App` metapac
 * Run the following command to get the SignalR client library by using LibMan.
 
   ```console
-  libman install @aspnet/signalr -p unpkg -d wwwroot/lib/signalr --files dist/browser/signalr.js --files dist/browser/signalr.min.js
+  libman install @aspnet/signalr@next -p unpkg -d wwwroot/lib/signalr --files dist/browser/signalr.js --files dist/browser/signalr.min.js
```

Comment:
I'd say my only issue is that the libman commands seem complex (sometimes excruciatingly so). I'd almost be interested in a libman-less alternative, wherein we just show the customer how to use npm natively to install the package(s) and then to move the files around. I know libman aims to make this easier but I'd ask that we think about native solutions too. Not a big deal for this particular article.

Reply:
@scottaddie see comments from @bradygaster on use of libman.  I believe we discussed this earlier, do you remember why we ended up sticking with libman?  If we think libman isn't ideal for this tutorial, now is a good time to make the change.

On the package name change from aspnet to microsoft, since that's just a week away now, I'm inclined to let the PR sit until that takes effect, then update the name and merge the PR.

Reply:
@tdykstra We used LibMan because Damian has asked that it be used in the project templates. See https://github.com/aspnet/AspNetCore/issues/5415.

My opinion is the command-line usage here is expected by VS Code users. If we were to ask VS users to run this command, then I'd be concerned.